cmd/operator-sdk/bundle/cmd.go

```diff
@@ -33,11 +33,16 @@ func NewCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "bundle",
 		Short: "Work with operator bundle metadata and bundle images",
-		Long: `Generate operator bundle metadata and build operator bundle images, which
-are used to manage operators in the Operator Lifecycle Manager.
+		Long: `
+Bundle metadata and build operator bundle images are used to manage operators in the OLM. 
```

Comment:
The previous description was more clear IMO.

Reply:
I got your point, but this is the desc of the bundle command and not just create because of this the goal of the changes here is to clarify its better. 

Anyway, your review was addressed here.